r1cs/gadgets/std/src/fields/mod.rs

```diff
@@ -551,7 +551,9 @@ pub(crate) mod tests {
     }
 
     #[allow(dead_code)]
-    pub(crate) fn from_bits_fp_gadget_test<ConstraintF: PrimeField>() {
+    // if little_endian is true (resp. false), then the reconstruction of a field element from its
+    // little (resp. big) endian bit representation is tested
+    pub(crate) fn from_bits_fp_gadget_test<ConstraintF: PrimeField>(little_endian: bool) {
```

Comment:
We can test both a priori, without the need to decide it by passing an additional flag

Reply:
I preferred to introduce the flag instead of writing two different functions to avoid code duplication, as most of the code is the same. Then the function is called twice with different values of the flag, so both big and little endian methods are tested. Is it ok or you prefer to have two distinct functions?

Reply:
Nono I simply meant let's test both BE and LE inside the same function anyway and remove the flag. We will always test both

Reply:
Ok, I added another private function that has the boolean flag to still avoid code duplication, removing the flag parameter from the `from_bits_fp_gadget_test`, so that a single call to such function allows to test both BE and LE. Let me know in case you think another solution is preferable.

Reply:
LGTM !